docs/tutorial/kymotracking.rst

```diff
@@ -252,3 +252,85 @@ The candidate with the lowest score is then selected.
 Since this algorithm is specifically looking for curvilinear structures, it can have issues with structures that are
 more blob-like (such as short-lived fluorescent events) or diffusive traces, where the particle moves randomly rather
 than in a uniform direction.
+
+
+Studying diffusion processes
+----------------------------
+
+To study diffusive processes we can make use of the Mean Squared Displacement (MSD).
+There are multiple ways to estimate this quantity.
+We use the following estimator:
+
+.. math::
+
+    \hat{\rho}[n] = \frac{1}{N - n} \sum_{i=1}^{N-n}\left(x_{i+n} - x_{i}\right)^2
+
+where :math:`\hat{\rho}[n]` corresponds to the estimate of the MSD for lag :math:`n`, :math:`N` reflects the number of samples, :math:`n` represents the lag we are computing and the trace positions at time frame i is given by :math:`x_i`.
+
+The reason we use this definition is because it results in the largest number of displacements being used thereby resulting in a well averaged estimate. The downside of this estimator however, is that it suffers from providing very correlated estimates :cite:`qian1991single,michalet2010mean,michalet2012optimal` which needs to be taken into account when doing subsequent analyses.
+
+In the following, we'll use three simulated :class:`~lumicks.pylake.kymotracker.kymoline.KymoLine` instances of length 62 based on a diffusion constant of `10.0`.
+
+With Pylake, we can calculate the MSD from a :class:`~lumicks.pylake.kymotracker.kymoline.KymoLine` with a single command::
+
+    kymolines[0].msd()
+
+This returns a tuple of lags and MSD estimates. If we only wish MSDs up to a certain lag, we can provide a `max_lag` argument::
+
+    >>> kymolines[0].msd(mag_lag = 5)
+    (array([0.16, 0.32, 0.48, 0.64, 0.8 ]), array([12.48593965, 16.34844311, 17.21359513, 27.25210869, 32.34473104]))
+
+MSDs are typically used to calculate diffusion constants. With pure diffusive motion (a complete absence of drift), 1-dimensional MSDs can be fitted by the following relation:
+
+.. math::
+
+    \rho[n] = 2 D t + offset
+
+where :math:`D` represents the diffusion constant, :math:`t` represents time and the offset is determined by the localization accuracy and is given by:
```

Comment:
👍

Reply:
oy, yes I have no idea why I typed 'cm'

I think it would be good to have the units in the docs because otherwise the user has to go through the code to see what's actually being done in order to figure out the units themselves, which kind of defeats the point of offering the feature in the first place a bit. 

It's definitely something we'll have to keep an eye on in the future, but since the units of the input values (um and sec) are assumed, there should be a definite unit on the return value